### DISCUSSION TITLE
Multiple selection of images not working anymore?

### DISCUSSION THREAD

**Discussion by svho12** [LycheeOrg/Lychee#2214] [Q&A] [answered]
Since version 5.x, it no longer seems possible to select several images at once, for example to delete or move them. As a result, images from the "unsorted" album, for example, can no longer be moved to another album in one step and have to be selected individually, which is not very user-friendly if there are a lot of images.

Is this option no longer available in v5 or have I overlooked how it works now?

In version 4, it was possible to select multiple images by holding down the Shift key and then right-click to trigger actions on multiple images.

Thanks for your great work!

**Comment by ildyria** [comment]
CTRL + click & Shift + click are still available.
I haven't had time to implement the click drag for selection.

**Reply by svho12** [reply]
Thanks for the hint. Unfortunately this only works partially and depends on which browser I am using. In Safari (I use MacOS) I can click on an image with the CTRL key pressed, which is then marked with a blue frame and the Lychee context menu opens at the same time. I can then select further images by holding down the Shift or Ctrl key.

However, I cannot, for example, select a series of images directly by holding down the Shift key. I always have to select a single image first by holding down the CTRL key. You can then select several images by pressing the Shift key.

But this only works in Safari. In Chrome or Firefox, the CTRL or Shift key does not change the behavior of the web interface and the images cannot be selected in this way.

**Reply by ildyria** [reply]
Apple >_<

**Reply by ildyria** [reply, marked as answer]
@svho12 I mostly work on Linux so those are kinda hard to debug... 

Nevertheless, you will be pleased to know that I just bought a magic mouse for the old 2012 Apple mac mini that my father gave me. I should be able to do some tests on mac Os & Safari and debug this further. I can't give you a timeline though.

In the mean time, I have sadly to suggest you to revert to the v4 layout by setting `LIVEWIRE_ENABLED=false` in your `.env`
(Version 4 and Version 5 are mostly switchable by setting a flag).

The Version 4 should be able to run with Lychee for quite a while as we are not going to remove the REST API.
However all development on that side of the front-end (full js without framework) are dead. All new shiny things will be happening on the new front-end (e.g. upload by chunk, etc).

**Reply by svho12** [reply]
Thank you for the workaround. The old frontend currently works much better for me than the new one.
With Chrome or Firefox on the Mac, everything works again without any problems.
It also works even better on the smartphone.

I'm glad to hear that you're also keeping an eye on the Mac.
Please let me know if I can help with testing on the Mac.

**Comment by ildyria** [comment]
Should be fixed hopefully on Version 6. :)
